utils/PtEtaPhiMLorentzVectorUtilities.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
```

Comment:
This entire big file, with code repetition with what already exists, is only needed to do:
```
        jets = physicsObj.get_jets(events, jet_collection_name)
        jet_pf_cands = physicsObj.get_jet_pf_cands(events, jet_collection_name, constituent_collection_name
```
I created it at some point to make things simpler, but it only makes everything more complicated! I do not think this file is needed at all (and some functions are duplicated and it is not compatible with TreeMaker...).

Reply:
removed file, copied only necessary functions